src/attributes.jl

```diff
@@ -910,6 +910,24 @@ A model attribute for the number of results available.
 """
 struct ResultCount <: AbstractModelAttribute end
 
+"""
+    ConflictStatus(result_index)
+    ConflictStatus()
+
+A model attribute for the `TerminationStatusCode` explaining why the conflict
+refiner stopped when computing the conflict `result_index`.
```

Comment:
There may be several minimal sets, even though as far as I know only Xpress can compute several IISes. (See https://www.msi-jp.com/xpress/learning/square/optimizer-2015.pdf, p. 38: `XPRSiisfirst` and `XPRSiisnext`.)

Reply:
Then it might help to have an equivalent of `ResultCount` for conflict. Also, as there is no relation between the index of the result and the index of the conflict, it might be better to use another name than `result_index`. `ConflictCount` and `conflict_index` maybe ?

Reply:
Indeed, for `ResultCount`. For `result_index`, I just followed @joaquimg and https://github.com/JuliaOpt/MathOptInterface.jl/pull/999. An IIS can be still thought of as a result of some computation, so I was wondering if it would not ease some parts of the code to keep the same field name?

Reply:
I'm hesitant. @odow what's your take on this ?

Reply:
Do we need to add multiple conflicts? It seems that this is a pretty niche part of the API. I vote just one for now, we can always add more later. If people are wanting to look at multiple IISs, they should use the solver API directly.